tests/CarbonInterval/ConstructTest.php

```diff
@@ -235,4 +235,9 @@ public function testInstanceWithDaysThrowsException()
     {
         CarbonInterval::instance(Carbon::now()->diff(Carbon::now()->addWeeks(3)));
     }
+
+    public function testCallInvalidStaticMethod()
+    {
+        $this->assertNull(CarbonInterval::anything());
```

Comment:
I think this is fine but interesting looking back we don't throw an exception  or something.